kubernetes/README.md

```diff
@@ -0,0 +1,17 @@
+# Kubernetes manifests
+
+These serve mostly as an example. You will probably want to use kustomize or some other configuration/deployment tool.
+
+## Workload type
+
+We recommend using a StatefulSet instead of deployment to mitigate high churn in long term metrics.
+Predictable pod name which in the instance label makes it easier for Prometheus to calculate the SLO over long time
+periods.
```

Comment:
```suggestion
We recommend using a StatefulSet instead of deployment to mitigate high churn in long term metrics - statefulset' strict pod name format (which is propagated to Prometheus' instance label) makes it easier for Prometheus to calculate the SLO over long time periods.
```

Reply:
Personally, `predictable` describes the StatefulSet pod name pattern better than `strict` but I might be wrong
Looking at the k8s docs, they use word `stable` :thinking:

Reply:
```suggestion
We recommend using a StatefulSet instead of deployment to mitigate high churn in long term metrics - statefulset' stable pod name format (which is propagated to Prometheus' instance label) makes it easier for Prometheus to calculate the SLO over long time periods.
```